mypy.ini

```diff
@@ -107,6 +107,7 @@ files = src/sentry/analytics/,
         src/sentry/tasks/symbolication.py,
```

Comment:
Ah, interesting, so to be clear do all type-checked files need to be included here?